### PR TITLE
mitmweb: add page, mitmproxy: update page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 9.0.1 (both commands)**

### CODE REVIEW

File: pages/common/mitmproxy.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # mitmproxy
 
 > An interactive man-in-the-middle HTTP proxy.
-> More information: <https://mitmproxy.org>.
+> More information: <https://docs.mitmproxy.org/stable/concepts-options>.
 
 - Start mitmproxy with default settings:
```
Comment (acuteenvy) [inline pages/common/mitmproxy.md:6]: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases
```suggestion
- Start `mitmproxy` with default settings:
```

File: pages/common/mitmproxy.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 
 `mitmproxy`
 
 - Start mitmproxy bound to custom address and port:
```
Comment (acuteenvy) [inline pages/common/mitmproxy.md:10]:
```suggestion
- Start `mitmproxy` bound to a custom address and port:
```

File: pages/common/mitmproxy.md [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 
-`mitmproxy -b {{ip_address}} -p {{port}}`
+`mitmproxy --listen-host {{ip_address}} --listen-port {{port}}`
+
+- Start mitmproxy using a script to process traffic:
```
Comment (acuteenvy) [inline pages/common/mitmproxy.md:14]:
```suggestion
- Start `mitmproxy` using a script to process traffic:
```

File: pages/common/mitmweb.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# mitmweb
+
+> A web-based interactive man-in-the-middle HTTP proxy.
+> See also: `mitmproxy`.
+> More information: <https://docs.mitmproxy.org/stable/concepts-options>.
+
+- Start mitmweb with default settings:
```
Comment (acuteenvy) [inline pages/common/mitmweb.md:7]:
```suggestion
- Start `mitmweb` with default settings:
```

File: pages/common/mitmweb.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+`mitmweb`
+
+- Start mitmweb bound to custom address and port:
```
Comment (acuteenvy) [inline pages/common/mitmweb.md:11]:
```suggestion
- Start `mitmweb` bound to a custom address and port:
```

File: pages/common/mitmweb.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+`mitmweb --listen-host {{ip_address}} --listen-port {{port}}`
+
+- Start mitmweb using a script to process traffic:
```
Comment (acuteenvy) [inline pages/common/mitmweb.md:15]:
```suggestion
- Start `mitmweb` using a script to process traffic:
```

File: pages/common/mitmproxy.md [hunk 7]
```diff
@@ -1,12 +1,16 @@
 # mitmproxy
 
 > An interactive man-in-the-middle HTTP proxy.
```
Comment (acuteenvy) [inline pages/common/mitmproxy.md:3]:
```suggestion
> An interactive man-in-the-middle HTTP proxy.
> See also: `mitmweb`.
```

**Review by acuteenvy — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**